charts/nri-prometheus/templates/clusterrolebinding.yaml

```diff
@@ -2,15 +2,15 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ template "nri-prometheus.fullname" . }}
+  name: {{ template "newrelic.common.naming.fullname" . }}
```

Comment:
I'm not sure if we have any preference using `template` or `include`, but if we did, this would be a go chance to go in one direction or another. WDYT?

Reply:
ups totally miss this , I think it should be [include](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-include-function)  Also the common library examples are with include. I will fix this on all the chart, thanks

Reply:
done added in new commit

Reply:
👍 thanks!